site/docs/spec.md

```diff
@@ -651,5 +651,3 @@ This serialization scheme is for storing single values as individual binary valu
 | **`struct`**                 | Not supported                                                                                                |
 | **`list`**                   | Not supported                                                                                                |
 | **`map`**                    | Not supported                                                                                                |
-
-
```

Comment:
Can you remove unnecessary whitespace changes to reduce commit problems?

Reply:
Done